tests/test_simulation.py

```diff
@@ -24,8 +24,9 @@ def _wait_until_passed_pending(simulation_handle):
 
 def _check_job_status_running(job):
     assert job['status'] == 'RUNNING'
-    print('-', job['status'], '-', round(
-        job['progress']['percentComplete'], 2), '%')
+    if job['status'] == 'RUNNING':
```

Comment:
Can you elaborate on this change?  This seems to break the intended functionality of the function (assuming the name is still accurate: `_check_job_status_running`)

Reply:
Honestly I'm not sure "what is up with that"

Reply:
reverted